cloud-native-declarative-osi-principles.md

```diff
@@ -1,44 +1,44 @@
 
-Network infrastructure can be separated into the underlying network fabric and the application and workload network. The underlying network fabric consists of the provisioning and configuration that resides at the lower OSI layers such as the implementation of the physical or virtual OSI layer 1 \(physical media, interconnects[27] such as buses[28] and layer 1 switches[29],[30],[31],[32], network adapters[33],[34], and other mechanisms[35],[36]\) or physical or virtual layer 2[37] \(layer 2 switches, Bridges[38] etc\). The application network functionality is deployed onto the underlying infrastructure network.
+Network infrastructure can be separated into the underlying network fabric \(underlay\) and the application or workload network \(overlay\). The establishment of an underlay network consists of the provisioning and configuration that resides at the lower OSI layers, such as the implementation of the physical or virtual OSI layer 1 \(physical media, interconnects[27] such as buses[28] and layer 1 switches[29],[30],[31],[32], network adapters[33],[34], and other mechanisms[35],[36]\) or physical or virtual layer 2[37] \(layer 2 switches, Bridges[38] etc\). The application \(overlay\) network functionality is deployed onto the underlying infrastructure network.
 
 _**L2**_ _- If a pipeline provisions_ _**physical network layer 1 or layer 2 infrastructure**, it will be provisioned_ _**immutably**._
 
-Configuration that relates layer 1 and layer 2 tend to be operating system boot options, machine specific bios settings \(e.g. SR-IOV bios settings\), or a configuration file for traditional layer 1 devices. These settings can be for physical hardware or virtual hardware. In order to provision physical hardware immutably it must be taken offline, reset to a known state \(e.g. networking device should get a ‘flashed’, complete replacement for its artifact updates\), have any of its patches applied sequentially. At this point new configuration changes can be applied and then the network infrastructure element is ready for use.
+Configuration that relates layer 1 and layer 2 tend to be operating system boot options, machine specific bios settings \(e.g. SR-IOV bios settings\), or a configuration file for traditional layer 1 devices. These settings can be for physical hardware or virtual hardware. In order to provision physical hardware immutably it must be taken offline, reset to a known state \(e.g. networking device should get a ‘flashed’ or 'pushed', complete replacement for its artifact updates\), have any of its patches applied sequentially. At this point new configuration \(that is stored in a templated format and is maintained in a version control system\) can be applied with the network infrastructure element then being ready for use. Both artifacts and configuration should be maintained independently of the physical or virtual device.
```

Comment:
> is maintained in a version control system

Should probably actually be "source" control system to more closely align with the immutable infra principles verbiage.